### ISSUE TITLE
large docker image on ec2

### ISSUE THREAD

**Issue by akeshavan** [nrdg/cloudknot#158]
I'm trying to run a large docker image (18GB) and the default AMI docker container size limit is 10GB -- how do I increase this?

**Comment by richford** [comment]
I think [this post](https://aws.amazon.com/premiumsupport/knowledge-center/increase-default-ecs-docker-limit/) may resolve this issue. i.e. create the file `/etc/sysconfig/docker-storage-setup` with contents:
```
#cloud-boothook
#!/bin/bash

cloud-init-per once docker_options echo 'OPTIONS="${OPTIONS} --storage-opt dm.basesize=20G"' >> /etc/sysconfig/docker
```

There's also [this article](https://aws.amazon.com/blogs/compute/building-high-throughput-genomic-batch-workflows-on-aws-batch-layer-part-3-of-4/). Scroll down to the section titled "Creating the custom AMI for AWS Batch" and it'll walk you through (1) attaching a scratch volume and (2) modifying the `/etc/fstab` file so that the scratch volume is mounted to `/docker_scratch` on system start.

**Comment by richford** [comment]
This should be resolved with #218.
